### PR TITLE
Height 28153 Fix

### PR DESCRIPTION
**Pull request by oten91**
Related to #1382

### CODE REVIEW

File: x/gov/keeper/subspace.go [hunk 1]
```diff
@@ -175,8 +177,11 @@ func (k Keeper) ModifyParam(ctx sdk.Ctx, aclKey string, paramValue []byte, owner
 	if err := k.VerifyACL(ctx, aclKey, owner); err != nil {
 		return err.Result()
 	}
-	if !k.cdc.IsAfterSecondUpgrade(ctx.BlockHeight()) && aclKey == "pos/MaxValidators" {
-		return types.ErrUnauthorizedHeightParamChange(types.ModuleName, codec.UpgradeHeight, aclKey).Result()
+
+	if ctx.BlockHeight() >= maxValidatorChangeAllowedHeight {
+		if !k.cdc.IsAfterSecondUpgrade(ctx.BlockHeight()) && aclKey == "pos/MaxValidators" {
```
Comment (Olshansk) [inline x/gov/keeper/subspace.go]: NITs:

1. Move `pos/MaxValidators` into a local non constant next to `maxValidatorChangeAllowedHeight`.
2. The name `IsAfterSecondUpgrade` is very clear int the context of codec. Note to self to have better naming for things like this in v1.

File: x/gov/keeper/subspace.go [hunk 2]
```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/pokt-network/pocket-core/x/gov/types"
 )
 
+const maxValidatorChangeAllowedHeight = 40000
```
Comment (Olshansk) [inline x/gov/keeper/subspace.go]: The changelog reference height `28153` and the release notes refer to `45353`. Is this just kind of an arbitrary number in between?

File: x/gov/keeper/subspace.go (hunk 1 above)
Comment (oten91) [reply x/gov/keeper/subspace.go]: 1-Refactoring done.
2-Naming is changed on the WIP branch for 0.8.0 as the upgrade mechanism is reworked/enhanced.

File: x/gov/keeper/subspace.go (hunk 2 above)
Comment (oten91) [reply x/gov/keeper/subspace.go]: Arbitrary indeed, just a height that  works for testnet and mainnet.

File: x/gov/keeper/subspace.go [hunk 3]
```diff
@@ -175,8 +178,12 @@ func (k Keeper) ModifyParam(ctx sdk.Ctx, aclKey string, paramValue []byte, owner
 	if err := k.VerifyACL(ctx, aclKey, owner); err != nil {
 		return err.Result()
 	}
-	if !k.cdc.IsAfterSecondUpgrade(ctx.BlockHeight()) && aclKey == "pos/MaxValidators" {
-		return types.ErrUnauthorizedHeightParamChange(types.ModuleName, codec.UpgradeHeight, aclKey).Result()
+
+	if ctx.BlockHeight() >= maxValidatorChangeAllowedMinHeight {
+
```
Comment (Olshansk) [inline x/gov/keeper/subspace.go:183]: Remove extra whiteline.

**Review by Olshansk — APPROVED** [review]

**Pull request merged**